tests/EalTest.cpp

```diff
@@ -136,6 +136,17 @@ TEST(SplineInterpTest, Extrapolate) {
                invalid_argument);
 }
 
+TEST(PolynomialTest, Evaluate) {
+  vector<double> coeffs = {1.0, 2.0, 3.0}; // 1 + 2x + 3x^2
+  EXPECT_EQ(2.0, eal::evaluatePolynomial(coeffs, -1, 0));
+}
+
+TEST(PolynomialTest, Derivatives) {
+  vector<double> coeffs = {1.0, 2.0, 3.0}; // 1 + 2x + 3x^2
+  EXPECT_EQ(-4.0, eal::evaluatePolynomial(coeffs, -1, 1));
+  EXPECT_EQ(6.0, eal::evaluatePolynomial(coeffs, -1, 2));
+}
+
```

Comment:
Add a test for `getVelocity`, including the error message if 3 vectors aren't supplied as input?

Reply:
Will do